samples/sqltransport/native-timeout-migration/SqlServer_3.1/Endpoint.Native/MigrateFromSql.sql

```diff
@@ -0,0 +1,24 @@
+﻿-- startcode MigrateFromSql
+
+declare @endpointName nvarchar(max) = 'PUT ENDPOINT NAME HERE';
+
+declare @endpointSchema nvarchar(max) = 'dbo';
+declare @dalayedTableSuffix nvarchar(max) = 'Delayed';
+declare @timeoutDataTable nvarchar(max) = REPLACE(@endpointName,'.','_') + '_TimeoutData';
+
+declare @migrateScript nvarchar(max);
+set @migrateScript = '
```

Comment:
When concatenating `nvarchar`s, it's probably a good idea to prefix all the literals with `N`:

`set @migrateScript = N'`